tests/Unit/Controllers/SettingsControllerTest.php

```diff
@@ -10,34 +10,237 @@
 use PHPUnit\Framework\MockObject\MockObject;
 use Symfony\Component\HttpFoundation\Session\Session;
 use Symfony\Component\HttpFoundation\Session\Storage\MockArraySessionStorage;
+use Psr\Log\LoggerInterface;
+use Engelsystem\Helpers\Authenticator;
+use Engelsystem\Test\Unit\HasDatabase;
+use Engelsystem\Http\Request;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Log\Test\TestLogger;
+use Engelsystem\Http\UrlGeneratorInterface;
+use Engelsystem\Http\UrlGenerator;
+use Engelsystem\Models\User\User;
 
 class SettingsControllerTest extends TestCase
 {
+    use HasDatabase;
+
+    /** @var Authenticator|MockObject */
+    protected $auth;
+
+    /** @var Config */
+    protected $config;
+
+    /** @var TestLogger */
+    protected $log;
+
+    /** @var Response|MockObject */
+    protected $response;
+
+    /** @var Request */
+    protected $request;
+
+    /** @var User */
+    protected $user;
+
+    /**
+     * @covers \Engelsystem\Controllers\SettingsController::password
+     */
+    public function testPassword()
+    {
+        /** @var Response|MockObject $response */
+        $this->response->expects($this->once())
+        ->method('withView')
```

Comment:
These should be indented by 4 spaces to be better readable, will have a look at how the linter can find those ;)